run_ci.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+sudo apt update
+sudo apt install -y python3 python3-venv python3-pip
```

Comment:
Considering we are mostly using macOS and Windows, would it be more straight forward to check if python is installed, and if not we echo instructions telling the user to go to Python's website and install python with the gui installer? Technically macOS can use homebrew, but it is not installed by default. Windows must use the gui installer.
I think telling users to run the installer themselves is also not as intrusive as demanding `sudo` and proceed to implicitly install a bunch of stuff of your machine.

Chatgpt gave me this bash function to detect Python's installation. It works on Git Bash (and WSL)
```
check_python_installed() {
    if command -v python3 &> /dev/null; then
        echo "Python is installed."
    else
        echo "Python is NOT installed."
        echo_python_installation_instructions
    fi
}
```